include/lsst/afw/image/VisitInfo.h

```diff
@@ -174,6 +174,10 @@ class VisitInfo : public table::io::PersistableFacade<VisitInfo>, public table::
     // get hour angle at the boresight
     geom::Angle getBoresightHourAngle() const;
 
+    // get parallactic angle at the boresight
+    // Equal to the angle measured at zenith between the North celestial pole and the boresight.
```

Comment:
This still does not specify the sign of the angle. Please add more words or reword. Here is one long, if clumsy, possibility: "This is the angular separation between two great circle arcs that meet at the zenith: the meridian and the arc from the zenith to the target. The angle is 0 if the target is on the meridian north of the zenith, positive if the object is east of the meridian, and pi if the object is on the meridian south of the zenith."

Also, use a /** style comment block when the doc is more than one line (as it now is, and justifiably so).